.prettierrc.yml

```diff
@@ -5,4 +5,4 @@ printWidth: 88
 proseWrap: always
 semi: true
 trailingComma: "es5"
-xmlWhitespaceSensitivity: "ignore"
+htmlWhitespaceSensitivity: "ignore"
```

Comment:
Why are you changing this?

Reply:
Because pre-commit issued the following warnings
![image](https://user-images.githubusercontent.com/17651502/89440369-e4c17e00-d721-11ea-9082-a9287741d18d.png)

Reply:
@Yajo do you know about this?

Reply:
Yes, those warnings are "correct" because the `xmlWhitespaceSensitivity` setting is only used when the XML prettier plugin is loaded, and prettier runs twice: once "normal" and once with the XML plugin loaded, so it's normal that the 1st time warns about this.
Just ignore them.